### DISCUSSION TITLE
Temporal

### DISCUSSION THREAD

**Discussion by mo6zes** [coollabsio/coolify#4408] [Service Template Requests]
https://docs.temporal.io/self-hosted-guide
Temporal is a durable execution platform that enables developers to build scalable applications without sacrificing productivity or reliability. The Temporal server executes units of application logic called Workflows in a resilient manner that automatically handles intermittent failures, and retries failed operations.

**Comment by unaimillan** [comment]
Agree, would love to see this service in the list of available "Resources" for deployment.

As far as I can see from the documentation, Temporal team have done a great job at simplifying this process.

Relevant links:
1) Official documentation on self-deployment \
https://docs.temporal.io/self-hosted-guide/deployment
2) Github repo with docker-compose.yaml files \
https://github.com/temporalio/docker-compose
